qcelemental/tests/test_units.py

```diff
@@ -73,23 +118,77 @@ def test_unit_conversion_nist(from_unit, to_unit, expected):
         ("au_permittivity", "au_charge**2 / (au_length * au_energy)", "1.0"),
         ("au_time", "au_action / au_energy", "1.0"),
         ("au_velocity", "au_length * au_energy / au_action", "1.0"),
-        ("au_pressure", "J / m^3", "2.94210e13"),  # note precision loss due to absence in CODATA
+        ("au_pressure", "J / m^3", "2.94210e13"),
     ],
 )
-def test_unit_conversion_other(from_unit, to_unit, expected):
+def test_unit_conversion_other_2014(from_unit, to_unit, expected):
+    # Build values and quantize to the tolerance
+    constants = qcelemental.PhysicalConstantsContext("CODATA2014")
```

Comment:
Is this slow rebuilding this context over and over again? It might be worth adding a small fixture for this.